server.js

```diff
@@ -40,8 +40,14 @@ app.use(addRequestId)
 // Add verbose logging of requests (see below)
 app.use(logRequests)
 
-// Require authentication for /extauth/backend/get-quote/ requests
-app.all('/extauth/backend/get-quote*', authenticate, function (req, res) {
+// Get authentication path from env, default to /extauth/backend/get-quote
+var authPath = '/extauth/backend/get-quote'
+if (process.env.AUTH_PATH) {
+  authPath = process.env.AUTH_PATH
+}
```

Comment:
I think output here would be helpful.
```javascript
console.log(`setting authenticated path to ${authPath}`)
```
Also might be good to update line 66 below, which still says `/backend/get-quote`.

Reply:
Makes sense. Fixed, thanks!